cmd/generate-fix/generate-fix.go

```diff
@@ -134,11 +134,11 @@ func main() {
 	internal.BuildGlobalFieldTypes(specs)
 
 	waitGroup.Add(1)
-	go genTags()
+	genTags()
```

Comment:
this intentional?

Reply:
Ah whoops, no. let me double check that file.

Reply:
Fixed